spec/controllers/idv/in_person/ssn_controller_spec.rb

```diff
@@ -30,16 +30,40 @@
 
   describe 'before_actions' do
     context('#confirm_in_person_address_step_complete') do
-      it 'redirects if the user hasn\'t completed the address page' do
-        # delete address attributes on session
-        flow_session[:pii_from_user].delete(:address1)
-        flow_session[:pii_from_user].delete(:address2)
-        flow_session[:pii_from_user].delete(:city)
-        flow_session[:pii_from_user].delete(:state)
-        flow_session[:pii_from_user].delete(:zipcode)
-        get :show
-
-        expect(response).to redirect_to idv_in_person_step_url(step: :address)
+      context 'residential address controller flag enabled' do
+        before do
+          allow(IdentityConfig.store).to receive(:in_person_residential_address_controller_enabled).
+            and_return(false)
```

Comment:
The context block does not match the before block.  
`residential address controller flag enabled` but  `in_person_residential_address_controller_enabled` returns false

Reply:
good catch!